specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/ManagedEnvironmentsDaprSubscriptions.json

```diff
@@ -0,0 +1,266 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-08-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/managedEnvironments/{environmentName}/daprSubscriptions": {
+      "get": {
+        "tags": [
+          "DaprSubscriptions"
+        ],
+        "summary": "Get the Dapr subscriptions for a managed environment.",
+        "operationId": "DaprSubscriptions_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
```

Comment:
The resource name parameter 'environmentName' should be defined with a 'pattern' restriction.